tgui/packages/tgui/interfaces/FlockPanel.js

```diff
@@ -0,0 +1,358 @@
+import { useBackend, useLocalState, useSharedState } from "../backend";
```

Comment:
whoever worked on this, put your preferred name and preferred license in the header (tagged similar to the other TGUI files)

Reply:
https://github.com/stonepillars/goon-flock/pull/330